pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -254,12 +257,15 @@ func (p *AnacondaInstallerISOTree) NewSquashfsStage() (*osbuild.Stage, error) {
 
 	switch p.anacondaPipeline.Type {
 	case AnacondaInstallerTypePayload, AnacondaInstallerTypeNetinst:
+		p.Stage2Image = "images/install.img"
 		squashfsOptions = osbuild.SquashfsStageOptions{
-			Filename: "images/install.img",
+			Filename: p.Stage2Image,
 		}
```

Comment:
Nitpick: the assignment now looks the same for all installer types, so it could be done just once after the `case` block.